.github/workflows/dockercompose-test.yml

```diff
@@ -0,0 +1,22 @@
+name: docker-compose validation
```

Comment:
Validation of docker-compose files. Only prints something and fails in case of issues with the YAML file.